Aliases.php

```diff
@@ -29,3 +29,12 @@ class ClaimGuidParser extends \Wikibase\DataModel\Statement\StatementGuidParser
 	class ClaimGuidParsingException extends \Wikibase\DataModel\Statement\StatementGuidParsingException {}
 
 }
+
+namespace Wikibase\DataModel {
+
+	/**
+	 * @deprecated since 3.0.0, use the base class instead.
+	 */
+	class StatementListProvider extends \Wikibase\DataModel\Statement\StatementListProvider {}
```

Comment:
Argh, you are right :astonished: Does `class_alias` actually work for interfaces? If this is the only issue, I will change this quickly on master.